### ISSUE TITLE
net-im/torchat 

### ISSUE THREAD

**Issue by TheChymera** [daktak/gentoo#2]
Hey there, I emerged your torchat ebuild, but I can't seem to find any installed launchers. How do I start torchat? Cheers!

**Comment by daktak** [comment]
Should show up in pidgin accounts.

**Comment by TheChymera** [comment]
As a protocol? I only see: AIM, Gtalk, Facebook, ICQ, IRC, MSN, Myspace, SIMPLE, XMPP, Yahoo, and Yahoo Japan.

**Comment by daktak** [comment]
Yeah should show up as a protocol,

can you equery f torchat and make sure you got libpurpletorchat.so? mine is in /usr/lib/purple-2

What version of pidgin are you using? I have  2.10.7-r4 

You could try copying that file to your ~/.purple/plugins dir...

Can you launch pidgin --debug and see if it says anything about torchat?

**Comment by TheChymera** [comment]
```
chymera@desktophost ~ $ equery f torchat
 * Searching for torchat ...
 * Contents of net-im/torchat-9999:
/usr
/usr/lib
/usr/lib/purple-2
/usr/lib/purple-2/libpurpletorchat.so
/usr/share
/usr/share/doc
/usr/share/doc/torchat-9999
/usr/share/doc/torchat-9999/README.markdown.bz2
/usr/share/doc/torchat-9999/README_DEBIAN.markdown.bz2

chymera@desktophost ~ $ eix -I pidgin
[I] net-im/pidgin
     Available versions:  2.10.7-r4 (~)2.10.7-r5 {aqua dbus debug doc eds gadu gnutls groupwise +gstreamer +gtk idn meanwhile mxit ncurses networkmanager nls perl prediction python sasl silc spell tcl tk +xscreensaver zephyr zeroconf PYTHON_SINGLE_TARGET="python2_7" PYTHON_TARGETS="python2_7"}
     Installed versions:  2.10.7-r5(22:24:23 31.08.2013)(dbus gnutls gstreamer gtk ncurses nls spell xscreensaver -aqua -debug -doc -eds -gadu -groupwise -idn -meanwhile -mxit -networkmanager -perl -prediction -python -sasl -silc -tcl -tk -zephyr -zeroconf PYTHON_SINGLE_TARGET="python2_7" PYTHON_TARGETS="python2_7")
     Homepage:            http://pidgin.im/
     Description:         GTK Instant Messenger client
```

So, do I have all I need?

**Comment by daktak** [comment]
Maybe you have the same issue?
https://github.com/prof7bit/TorChat/issues/12
